### PR TITLE
feature: enable XSLT transformations when exporting XML

### PR DESCRIPTION
**Pull request by dkarlovi**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #121 

Contains a bugfix which loaded import provider form types for export providers because of a typo.

TODO:
- [x] the usual asset handling support in UI
- [x] passing the asset path to the export process
- [x] if non-empty, swap the target path with a temporary one, build the command line and pipe to the target path

Sample:
```
# xsltFile (in config): web/var/assets/xslt/products.xslt
# temporary export to:  /tmp/temporary123.xml
# target export to:     web/var/assets/exports/products.xml
env xsltproc web/var/assets/xslt/products.xslt /tmp/temporary123.xml > web/var/assets/exports/products.xml 
```

### CODE REVIEW

File: src/ImportDefinitionsBundle/Resources/config/services/forms.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ services:
         arguments:
             - '%import_definitions.model.export_definition.class%'
             - '%import_definition.form.type.export_definition.validation_groups%'
-            - '@import_definition.form.registry.provider'
+            - '@import_definition.form.registry.export_provider'
```
Comment (dkarlovi) [inline src/ImportDefinitionsBundle/Resources/config/services/forms.yml]: Bugfix. :1st_place_medal:

File: src/ImportDefinitionsBundle/Resources/config/pimcore/config.yml [hunk 2]
```diff
@@ -61,4 +62,4 @@ import_definitions:
             fetcher_abstract: '/bundles/importdefinitions/pimcore/js/fetchers/abstract.js'
             fetcher_objects: '/bundles/importdefinitions/pimcore/js/fetchers/objects.js'
         css:
-            import_definition: '/bundles/importdefinitions/pimcore/css/importdefinition.css'
\ No newline at end of file
+            import_definition: '/bundles/importdefinitions/pimcore/css/importdefinition.css'
```
Comment (dkarlovi) [inline src/ImportDefinitionsBundle/Resources/config/pimcore/config.yml]: This project needs `.editorconfig` and `.gitattributes` :)

Comment (dpfaffenbauer) [reply src/ImportDefinitionsBundle/Resources/config/pimcore/config.yml]: PR? :)

**Comment by dkarlovi** [comment]
@dpfaffenbauer should be ready for review.

**Comment by dpfaffenbauer** [comment]
👍 thanks a lot

**Pull request merged**